megamek/data/mechfiles/mechs/3085u/ONN/Beowulf C.mtf

```diff
@@ -1,7 +1,6 @@
 Version:1.0
 Beowulf
 C
-mul id:8073
```

Comment:
Please don't delete MUL ID's. I'm part of the MUL team and often have access to the ID's prior to release.